sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs

```diff
@@ -1018,7 +1019,7 @@ private void EnsureClient()
                         return (await BlobRestClient.Blob.BreakLeaseAsync(
                             Pipeline,
                             Uri,
-                            breakPeriod: breakPeriodInSeconds,
+                            breakPeriod: serviceBreakPeriod,
```

Comment:
Does this need a `.TotalSeconds` or similar?

Reply:
`serviceBreakPeriod` is a local var where I've already converted to seconds.

Reply:
I'm not sure how I missed that...  but yes, yes it is.